app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -104,6 +107,18 @@ class HomeActivityTest {
       .inject(this)
   }
 
+  @Test
+  fun testNavigationDrawerTestActivity_clickNavigationDrawerHamburger_clickOnHeader_opensProfileProgressActivity() {
```

Comment:
Rename this test name to testHomeActivity.

Reply:
Updated.